docs/content.zh/docs/dev/datastream/user_defined_functions.md

```diff
@@ -77,16 +77,20 @@ data.reduce((i1,i2) -> i1 + i2);
 
 ```java
 class MyMapFunction implements MapFunction<String, Integer> {
-  public Integer map(String value) { return Integer.parseInt(value); }
-};
+  public Integer map(String value) {
+    return Integer.parseInt(value);
+  }
```

Comment:
let's move this back to a single line; also applies to other instances in this file, the english version, and I think 1 other page.